src/components/Details/BootOrder/BootOrder.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const BootOrder = props => {
+  const { bootableDevices } = props;
```

Comment:
Nitpicking: Rest of the code follows syntax:
```
const MyComponent = ({ bootableDevices }) => {};
```

Reply:
Fixed and rebased. =)